api/v1alpha1/openapi.yaml

```diff
@@ -4989,3 +5004,46 @@ components:
       required:
         - conditions
       type: object
+    Version:
+      required:
+        - major
+        - minor
+        - gitVersion
+        - gitCommit
+        - gitTreeState
+        - buildDate
+        - goVersion
+        - compiler
+        - platform
+        - patch
```

Comment:
This feels like too much information.  Can we have a single version string that will provide us the necessary information?

Reply:
Removed. Now pointers are generated, but that's ok

Reply:
also this is the info we're getting from cli, so service should be the same, so it's comparable

Reply:
The pointers are not a problem, but I'd prefer it to have the same format as cli's version